docker-compose.yml

```diff
@@ -2,6 +2,9 @@ version: "3"
 
 services:
   postgresql:
+    image: postgres
+    environment:
+      - POSTGRES_DB=postgres
```

Comment:
The database user has to be `pupa` so we don't want to reset the default user here.